runners/direct-java/src/test/java/org/apache/beam/runners/direct/StructuralKeyTest.java

```diff
@@ -53,6 +54,14 @@ public void getKeyNotSameInstance() {
   }
 
   @Test
+  public void emptyKeysNotEqual() {
+    StructuralKey<?> empty = StructuralKey.empty();
+
+    assertThat(empty, not(Matchers.<StructuralKey<?>>equalTo(StructuralKey.empty())));
```

Comment:
Yes, but this is the general pattern used in (almost) all of the other DirectRunner tests.